### DISCUSSION TITLE
我在mac的docker desktop中拉取了镜像vvbbnn00/warp-clash-api:latest，在容器运行之后，无法正常访问21001，请问这个是需要怎么配置呢？

### DISCUSSION THREAD

**Discussion by runping-wei** [vvbbnn00/WARP-Clash-API#155] [Q&A]
![image](https://github.com/vvbbnn00/WARP-Clash-API/assets/71248558/78ad7140-37e2-427b-af77-bab66bb6dfe0)
![image](https://github.com/vvbbnn00/WARP-Clash-API/assets/71248558/bf3f06e9-207a-4950-8382-ce615711ef15)
请问这种是哪里出错了呢，求解答！

**Comment by kilvn** [comment]
用 127.0.0.1 试下

**Reply by runping-wei** [reply]
还是一样的提示，说是拒绝了我们的连接请求。

**Comment by kilvn** [comment]
lsof -i:21001 先看下服务有没有运行吧。

**Reply by runping-wei** [reply]
请问这个要怎么看呢 小白 还不太会弄

**Comment by vvbbnn00** [comment]
在创建容器的时候似乎没有映射端口，如果是使用docker run来运行的话，可以尝试添加`-p 21001:3000`
